### PR TITLE
fix deprecation warning "explicit passing of coroutine objects to asyncio.wait()"

### PR DESCRIPTION
**Pull request by decaz**
## What do these changes do?

Fix `DeprecationWarning: The explicit passing of coroutine objects to asyncio.wait() is deprecated since Python 3.8, and scheduled for removal in Python 3.11.`

https://docs.python.org/3/library/asyncio-task.html?highlight=wait#asyncio-example-wait-coroutine

## Are there changes in behavior for the user?

Nope.

## Related issue number

No.

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -124,9 +124,9 @@ Client example
 
 
     tasks = (
-        get_mp3("server1.com", 21, "login", "password"),
-        get_mp3("server2.com", 21, "login", "password"),
-        get_mp3("server3.com", 21, "login", "password"),
+        asyncio.create_task(get_mp3("server1.com", 21, "login", "password")),
+        asyncio.create_task(get_mp3("server2.com", 21, "login", "password")),
+        asyncio.create_task(get_mp3("server3.com", 21, "login", "password")),
```
Comment (pohmelie) [inline README.rst:129]: This also should be changed in `docs/index.rst`.

Comment (pohmelie) [reply README.rst:129]: Oops, my bad. It is already changed.

**Comment by decaz** [comment]
@pohmelie can you release new version to avoid this warning, please?

**Comment by pohmelie** [comment]
Unfortunately something wrong with travis-ci, will try github actions tomorrow.

**Comment by decaz** [comment]
@pohmelie 🙏🏻

**Pull request merged**